File: backend/conexion/conexion.php
<?php

$server="localhost";
$user="root";
$pass="";
$db="test";

$conexion = new mysqli($server, $user, $pass, $db);

if ($conexion->connect_errno)
	die("Failure conexion " . $conexion->connect_errno );
else
	return $conexion;

?>

File: backend_python/api/conexion/conexion.py
# Equivalente a conexion.php
import mysql.connector

# Configuración de la base de datos
DB_CONFIG = {
	"host": "localhost",
	"user": "root",
	"password": "",
	"database": "test"
}

def get_connection():
	"""
	Crear la conexion esta en mySQL y no en SQLite por lo que cuidao
	"""
	try:
		conexion = mysql.connector.connect(**DB_CONFIG)
		return conexion
	except mysql.connector.Error as err:
		print(f"Failure conexion: {err.errno}")
		return None